client/state/jetpack/connection/reducer.js

```diff
@@ -47,7 +50,14 @@ export const requests = createReducer( {}, {
 	[ JETPACK_CONNECTION_STATUS_REQUEST_SUCCESS ]: createRequestReducer( false )
 } );
 
+export const disconnectsRequests = createReducer( {}, {
```

Comment:
I'd probably name this `disconnectRequests`, but that's definitely not a blocker.